### DISCUSSION TITLE
Équipements purchasing funding help

### DISCUSSION THREAD

**Discussion by papapapi1987** [community/community#135882] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Good morning everyone. My name is Festus Emmanuel and am a self thought and am having a problem and I've try solving it by myself but couldn't. My problem is, I used most of my savings to purchase the laptop which I used to start my learning journey but unfortunately it crached some months ago and since then I've been trying to get a better one but thing the country are so had right now that I just can't. I'm desperately in need of a laptop in order to continue my learning. Please I know this is inappropriate but am the point where I really want to resume learning before I start forgetting everything I learned please anyone assist me. Thanks in advance for your help and support.

**Comment by ebndev** [comment]
Hi @papapapi1987, I'm sorry you're going through that. 

Unfortunately, we currently do not allow self-promotion or solicitation on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thanks for understanding.
